src/containers/ProjectMonitoring.js

```diff
@@ -0,0 +1,5518 @@
+import React from 'react'
+import Flexbox from 'flexbox-react'
+import { connect } from 'react-redux'
+import GraphHeartbeats from '../components/GraphHeartbeats'
+import GraphHttpRequests from '../components/GraphHttpRequests'
+import GraphOsData from '../components/GraphOsData'
+import GraphUrls from '../components/GraphUrls'
+import OsDataWidget from '../components/OsDataWidget'
+import HttpRequestWidget from '../components/HttpRequestWidget'
+
+const mapStateToProps = state => ({
+  session: state.session,
+  projects: state.projects,
+  projectMembers: state.projectMembers
+})
+
+const mapDispatchToProps = {
```

Comment:
remove this if you are not using it. You can just replace it with `null` in the connect function.